.ci/scripts/ecr-images.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+IMAGE_SUFFIX=$1
+ECR_REGISTRY=$2
+ELASTIC_AGENT_DOCKER_TAG=$(make get-ci-agent-version)
```

Comment:
nit: I suggest improving the variable naming
pushing to container repository doesn't necessarily means ECR
`ECR_REGISTRY` -> `CONTAINER_REPO`

Reply:
fixed -> CONTAINER_REGISTRY